exasol/ds/sandbox/lib/ansible/ansible_access.py

```diff
@@ -1,7 +1,9 @@
 import ansible_runner
```

Comment:
Having a module `ansible_access` which just contains a class `AnsibleAccess` does not make much sense and only increases noise and verbosity imho. (`ansible_access.AnsibleAcces` vs `ansible.Access` )

`ansible.Access` should be as descriptive.

Reply:
You're speaking from my soul, Nico.
See my (removed) comment in [commit 5e66b3405](https://github.com/exasol/data-science-sandbox/pull/40/commits/5e66b34051f9783f3f41c4985762959a2720fab6#diff-6870d7bc222dcc2be29df4ec484c7bd1ca0b666a7ace84497069a3b57c3ddcde).

Could we address the renaming in the scope of ticket #65 and I'd love to kick it off in a pair-discussion?